examples/omni_multiple.ipynb

```diff
@@ -0,0 +1,253 @@
+    "    solvent_indices = get_selection_indices(frame, \"resname HOH\")\n",
+    "    \n",
+    "    selection = RenderingSelection(\"solvent\", \"Solvent\")\n",
+    "    selection.set_particles(solvent_indices)\n",
+    "    selection.hide = True\n",
+    "    selection.interaction_method = \"none\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2019-06-06T16:04:25.397984Z",
+     "start_time": "2019-06-06T16:04:25.395015Z"
+    },
+    "extensions": {
+     "jupyter_dashboards": {
+      "version": 1,
+      "views": {
+       "grid_default": {
+        "hidden": true
+       },
+       "report_default": {
+        "hidden": true
+       }
+      }
+     }
+    }
+   },
+   "outputs": [],
+   "source": [
+    "nanotube_xml = \"openmm_files/nanotube.xml\"\n",
+    "alanine_xml = \"openmm_files/17-ala.xml\"\n",
+    "\n",
+    "omni_nanotube = OpenMMSimulation.from_xml_path(nanotube_xml)\n",
+    "omni_alanine = OpenMMSimulation.from_xml_path(alanine_xml)\n",
+    "\n",
+    "nanotube_selection = RenderingSelection(SELECTION_ROOT_ID)\n",
+    "nanotube_selection.renderer = {\n",
+    "    'render': 'ball and stick',\n",
+    "    'color': {'type': 'particle index', 'gradient': [\"Orange\", \"Blue\"]}\n",
+    "}\n",
+    "\n",
+    "alanine_selection = RenderingSelection(SELECTION_ROOT_ID)\n",
+    "alanine_selection.renderer = {\n",
+    "    'color': 'Gold',\n",
+    "    'scale': 0.1,\n",
+    "    'render': 'liquorice'\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2019-06-06T16:04:25.493146Z",
+     "start_time": "2019-06-06T16:04:25.400442Z"
+    },
+    "extensions": {
+     "jupyter_dashboards": {
+      "version": 1,
+      "views": {
+       "grid_default": {
+        "col": 8,
+        "height": 12,
+        "hidden": true,
+        "row": 0,
+        "width": 4
+       },
+       "report_default": {
+        "hidden": false
+       }
+      }
+     }
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ragzo: NanoVer iMD Server: serving at [::]:63673\n"
+     ]
+    }
+   ],
+   "source": [
+    "# This will just automatically close previous instances of the runner if you re-run this cell multiple times\n",
+    "try:\n",
+    "    omni_runner.close()\n",
+    "except NameError:\n",
+    "    pass\n",
+    "omni_runner = OmniRunner.with_basic_server(port=0)\n",
+    "\n",
+    "omni_runner.add_simulation(omni_nanotube)\n",
+    "omni_runner.set_simulation_selections(\n",
+    "    omni_nanotube, \n",
+    "    nanotube_selection, \n",
+    "    make_solvent_hiding_selection(omni_nanotube)\n",
+    ")\n",
+    "\n",
+    "omni_runner.add_simulation(omni_alanine)\n",
+    "omni_runner.set_simulation_selections(omni_alanine, alanine_selection)\n",
+    "\n",
+    "omni_runner.next()\n",
```

Comment:
`.next()` seems to immediately load the second simulation, rather than loading the first one.

Is it also worth adding a cell with `omni_runner.next()` to allow the client to run the cell multiple times to see the simulations switching between the two loaded simulations with the pre-defined selections?